Gemfile

```diff
@@ -17,129 +17,129 @@ gem 'uglifier', '>= 1.3.0'
 # Use CoffeeScript for .coffee assets and views
 gem 'coffee-rails', '~> 4.2'
 
-gem 'therubyracer', platforms: :ruby
+gem 'therubyracer', '~> 0.12.3', platforms: :ruby
 
 # Use jquery as the JavaScript library
-gem 'jquery-rails'
+gem 'jquery-rails', '~> 4.3.1'
 # Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
 gem 'turbolinks', '~> 5'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 gem 'jbuilder', '~> 2.5'
 # Use autoprefixer to avoid writing css prefixes
-gem 'autoprefixer-rails'
+gem 'autoprefixer-rails', '~> 7.1.4.1'
 
-gem 'bootstrap-sass'
+gem 'bootstrap-sass', '~> 3.3.7'
 
-gem 'carrierwave'
+gem 'carrierwave', '~> 1.1.0'
 
 # Authentication
-gem 'devise'
+gem 'devise', '~> 4.3.0'
 gem 'devise-async', '~> 0.7.0'
 # devise-i18n support
-gem 'devise-i18n'
+gem 'devise-i18n', '~> 1.2.0'
 
-gem 'responders'
+gem 'responders', '~> 2.4.0'
 
-gem 'active_model_serializers'
+gem 'active_model_serializers', '~> 0.10.6'
 
-gem 'foreman'
+gem 'foreman', '~> 0.84.0'
 
 # Active Admin
 gem 'activeadmin', git: 'https://github.com/activeadmin/activeadmin'
 gem 'inherited_resources', git: 'https://github.com/activeadmin/inherited_resources'
 
 # Enables Slim templates
-gem 'slim-rails'
+gem 'slim-rails', '~> 3.1.2'
 
 # Sidekiq
-gem 'sidekiq'
-gem 'sidekiq-failures'
-gem 'sidekiq_mailer'
-gem 'sinatra', require: nil
+gem 'sidekiq', '~> 5.0.4'
+gem 'sidekiq-failures', '~> 1.0.0'
+gem 'sidekiq_mailer', '~> 0.0.8'
+gem 'sinatra', '~> 2.0.0', require: nil
 
 # Authorization Policies
-gem 'pundit'
+gem 'pundit', '~> 1.1.0'
 
 # Exceptions Report
-gem 'rollbar'
+gem 'rollbar', '~> 2.15.4'
 
 # SEO Meta Tags
-gem 'meta-tags'
-gem 'metamagic'
+gem 'meta-tags', '~> 2.6.0'
+gem 'metamagic', '~> 3.1.7'
 
-gem 'newrelic_rpm'
+gem 'newrelic_rpm', '~> 4.4.0.336'
 
-gem 'recipient_interceptor'
+gem 'recipient_interceptor', '~> 0.1.2'
 # CORS support
 gem 'rack-cors', '~> 1.0.2', require: 'rack/cors'
 
 # Use for DoS attacks
-gem 'rack-attack'
+gem 'rack-attack', '~> 5.0.1'
 
 gem 'webpacker', '~> 3.0'
 
 group :development do
   # Gem to detect N+1 queries
-  gem 'better_errors'
-  gem 'bullet'
+  gem 'better_errors', '~> 2.3.0'
+  gem 'bullet', '~> 5.6.1'
   gem 'listen', '~> 3.0.5'
   # Access an IRB console on exception pages or by using <%= console %> anywhere in the code.
-  gem 'web-console'
+  gem 'web-console', '~> 3.5.1'
   # Spring speeds up development by keeping your application running in the background.
   # Read more: https://github.com/rails/spring
-  gem 'spring'
+  gem 'spring', '~> 2.0.2'
+  gem 'meta_request', '~> 0.4.3'
```

Comment:
Gem `meta_request` should appear before `spring` in their gem group.

Check that gems should be displayed alphabetically within its group